### PR TITLE
Update base salary and geoscaling info

### PR DESCRIPTION
**Pull request by makew0rld**
Two updates to `salary.md`:

- Base salary in the formula is updated to $51,250 to match the salary table from #280 
- Geo-scaling description is corrected to what Ben informed me when I joined

### CODE REVIEW

**Comment by makew0rld** [comment]
@hyphacoop/operations please double check this as salary is involved

File: How-we-work/salary.md [hunk 1]
```diff
@@ -7,21 +7,21 @@ This document outlines the salary tiers of Hypha members and the review process.
...
+| Criteria           | Scaling factor  | Description                                                                             |
+|:-------------------|----------------:|:----------------------------------------------------------------------------------------|
+| **L**evel          |       `L=1.160` | Skills and experiences `l=[0-6]`                                                        |
+| **R**esponsibility |       `R=1.120` | Responsibilities `r=[0-4]`                                                              |
+| **S**eniority      |       `S=1.025` | Seniority in organization `s=[0-3]`                                                     |
+| Commitment         | `0.250`-`1.200` | Modifier applied to members working part-time or over-time each week                    |
+| Contractor         |         `1.050` | Modifier applied to contractors who do not receive employee benefits                    |
+| Geo-upscaling      |         `1.150` | Modifier applied to members living in higher cost of living regions compared to Toronto |
+| Geo-downscaling    |         `0.850` | Modifier applied to members living in lower cost of living regions compared to Toronto  |
```
[the remark this answers is not in the document]
Comment (makew0rld) [reply How-we-work/salary.md:23]: If we don't do geoscaling inside Canada then that should definitely be mentioned as well. I don't know what official policy is so I will wait for ops to chime in.

**Comment by LexaMichaelides** [comment]
This is great, thanks so much Cole and Vincent!

File: How-we-work/salary.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (LexaMichaelides) [reply How-we-work/salary.md:23]: I think it's okay to have Toronto in here since that really is our base case and is easily clarified when someone asks.

**Review by LexaMichaelides — APPROVED** [review]

**Comment by tripledoublev** [comment]
Thanks @LexaMichaelides

**Pull request merged**